### ISSUE TITLE
run gbp import-dsc --pristine-tar by default

### ISSUE THREAD

**Issue by pravi** [LeoIannacone/npm2deb#64]
Currently we have to manually import the dsc file to git. It would be nice if this step was automated by npm2deb.

**Comment by simevo** [comment]
this is what I do:
```
npm2deb create xxx
cd xxx
gbp import-dsc --author-is-committer --pristine-tar `find . | grep "dsc$"`
cd node-xxx
git tag -d debian/x.y.x-1
git commit --amend -m 'debian directory as generated by npm2deb'
cd ..
mv node-xxx ..
cd ..
rm -rf xxx
```

**Comment by shanavas786** [comment]
Closed by #149
